alibi_detect/cd/sklearn/classifier.py

```diff
@@ -232,3 +264,19 @@ def score(self, x: Union[np.ndarray, list]) -> Tuple[float, float, np.ndarray, n
         p_val, dist = self.test_probs(y_oof, probs_oof, n_ref, n_cur)
         probs_sort = probs_oof[np.argsort(idx_oof)]
         return p_val, dist, probs_sort[:n_ref, 1], probs_sort[n_ref:, 1]
+
+    def _score_rf(self, x: Union[np.ndarray, list]) -> Tuple[float, float, np.ndarray, np.ndarray]:
+        x_ref, x = self.preprocess(x)
+        x, y, _ = self.get_splits(x_ref, x, return_splits=False)
+        self.model.fit(x, y)
+        # it is possible that some inputs do not have OOB scores. This is probably means that too few trees were
+        # used to compute any reliable estimates. In this case shall we raise an error or keep going by
```

Comment:
Can probably only keep the first line of this comment since sklearn would throw the appropriate warning and the other 2 lines of the comment are more of a question for reviewers.